drivers/SmartThings/zigbee-window-treatment/profiles/window-treatment-powerSource.yml

```diff
@@ -0,0 +1,23 @@
+name: window-treatment-powerSource
+components:
+  - id: main
+    capabilities:
+      - id: windowShade
+        version: 1
+      - id: windowShadeLevel
+        version: 1
+      - id: windowShadePreset
+        version: 1
+      - id: battery
+        version: 1
+      - id: refresh
+        version: 1
+      - id: powerSource
```

Comment:
I'd recommend putting `powerSource` above of the `battery` capability.

Reply:
`powerSource` capability is placed above the `battery` capability.